war.rb

```diff
@@ -0,0 +1,10 @@
+module War
+    #  A country can win a war if no of states is more than 10, population is more than 20 crores and army strength is more than 10 lakh
+    def can_win_war(country)
```

Comment:
method name should be `can_win_war?` and it should return `true/false` based on that you may have shown the message